src/MessageTile/stories/messageTile.jsx

```diff
@@ -12,13 +11,11 @@ import by from './by.svg'
 import cc from './cc.svg'
 import nd from './nd.svg'
 
-import accentTheme from '../../styles/themes/accent'
-import coreTheme from '../../styles/themes/core'
 import defaultTheme from '../../styles/themes/default'
 
 export default () => (
   <ComponentOverview heading='MessageTile' component={UnwrappedMessageTile}>
-    <div style={{ width: 220 }}>
+    <div style={{ marginTop: 20, width: 220 }}>
```

Comment:
Where is this margin sourced from? This might be an error in Zeplin as we are aiming to stick with divisions of 8.

Reply:
These are examples only; to show how they could be used. The margins are only to separate the examples.

Reply:
I've adjusted this to 16px; may as well follow a 8px base unit if we're going to use it for everything else.